### PR TITLE
feedback on book styling

### PR DESCRIPTION
**Pull request by jeanetteclark**
@dlebauer asked me to have a look at the book and give it a review. I haven't had a chance to go through chapters in any detail (the last one in particular won't run for me) but I've made some commits here that will build the book, set the TOC styling, and make an attempt to fix some of the header levels in the chapters. 

more tomorrow

### CODE REVIEW

File: index.Rmd [hunk 1]
```diff
@@ -12,9 +12,6 @@ documentclass: book
 
 This book is intended to quickly introduce users to TERRA REF data through a series of tutorials. TERRA REF has many types of data, and most can be  accessed in multiple ways. Although this makes it more complicated to learn (and teach!), the objective is to provide users with the flexibility to access data in the most useful way.
 
-
-## Contents
```
Comment (jeanetteclark) [inline index.Rmd]: using the section collapsing in the table of contents will collapse all of the sections except for the ones in the index.Rmd (I'm not sure why honestly). I condensed the number of top level headers in this chapter to make the TOC styling look better

**Comment by KristinaRiemer** [comment]
I haven't had a chance to look at this in detail yet, but the book built with no errors the first time I tried, so that's really good! 

The table of contents has a greyed out "Section 1: Vignettes", should there also be a matching "Section 2: Tutorials between 6 and 7? I couldn't figure out how that was generated. 
screen shot 2019-02-05 at 3 42 01 pm

**Comment by jeanetteclark** [comment]
`00-introduction.Rmd` has the line that implements the first section

File: _bookdown.yml [hunk 2]
```diff
@@ -1,9 +1,16 @@
 book_filename: "terraref-tutorials"
+delete_merged_file: true
```
Comment (dlebauer) [inline _bookdown.yml]: I was wondering how to do that, thanks!

File: index.Rmd [hunk 3]
```diff
@@ -37,7 +33,7 @@ At a minimum, you should have:
 * Software:
     + Software requirements vary with the tutorials, and may be complex
 
-## User Accounts and permission to access TERRA REF data
+### User Accounts and permission to access TERRA REF data
 
 We have tried to write these tutorials using open access sample data sets. However, access to much of the data will require you to 1) fill out the TERRA REF Beta user questionaire ([terraref.org/beta](terraref.org/beta)) and 2) request access to specific databases.
```
Comment (dlebauer) [inline index.Rmd]:
```suggestion
The first few chapters in the 'vignettes' section use publicly available sample data sets.
Subsequent sections are also written to use publicly available data sets, but some of the examples require data that requires users to sign up.
To sign up, you will need to 1) fill out the TERRA REF Beta user questionaire ([terraref.org/beta](terraref.org/beta)) and 2) request access to specific databases.
```

File: index.Rmd [hunk 4]
```diff
@@ -64,7 +60,7 @@ The TERRA REF website: [terraref.org](http://terraref.org/)
 The TERRA REF Technical Documentation: [docs.terraref.org](docs.terraref.org)
 
 
-## Finding help
+### Finding help
 
 - Slack at terra-ref.slack.com ([signup](https://terraref-slack-invite.herokuapp.com/))
```
Comment (dlebauer) [inline index.Rmd]:
```suggestion
- Slack at terra-ref.slack.com ([signup](https://terraref-slack-invite.herokuapp.com/)).
```

File: sensors/01-meteorological-data.Rmd [hunk 5]
```diff
@@ -214,35 +217,29 @@ dir.create(output_dir, showWarnings = FALSE, recursive = TRUE)
...
-library(RCurl)
-library(ggplot2)
-
 # Get Spaces from Clowder - without authentication, result will be Sample Data
 spaces <- fromJSON(paste0(api_url, '/spaces'))
 print(spaces %>% select(id, name))
 ```
 
-```{r list-of-datasets}
+```{r list-of-datasets, eval = F}
```
Comment (dlebauer) [inline sensors/01-meteorological-data.Rmd]: per http://adv-r.had.co.nz/Style.html

```suggestion
```{r list-of-datasets, eval = FALSE}
```

File: sensors/01-meteorological-data.Rmd [hunk 6]
```diff
@@ -214,35 +217,29 @@ dir.create(output_dir, showWarnings = FALSE, recursive = TRUE)
...
 
 # Get list of (at most 20) Datasets within that Space from Clowder
 datasets <- fromJSON(paste0(api_url, '/spaces/', spaces$id, '/datasets'))
 print(datasets %>% select(id, name))
 ```
 
-```{r list-of-files}
+```{r list-of-files, eval = F}
```
Comment (dlebauer) [inline sensors/01-meteorological-data.Rmd]:
```suggestion
```{r list-of-files, eval = FALSE}
```

File: sensors/01-meteorological-data.Rmd [hunk 7]
```diff
@@ -214,35 +217,29 @@ dir.create(output_dir, showWarnings = FALSE, recursive = TRUE)
...
 ncfiles <- files[grepl('environmentlogger.nc', files$filename), ]
 print(ncfiles %>% select(id, filename))
 ```
 
-## Download netCDF 1/s data from Clowder
+#### Download netCDF 1/s data from Clowder
 
 
-```{r nc-download, echo=FALSE}
+```{r nc-download, echo=FALSE, eval = F}
```
Comment (dlebauer) [inline sensors/01-meteorological-data.Rmd]:
```suggestion
```{r nc-download, echo=FALSE, eval = FALSE}
```

File: sensors/01-meteorological-data.Rmd [hunk 8]
```diff
@@ -254,15 +251,11 @@ for (i in 1:length(sources)) {
...
+#### Using the netCDF 1/s data
 
 One use case getting the solar spectrum associated with a particular hyperspectral image.
 
-```{r}
-library(ncdf4)
-library(ncdf.tools)
-library(lubridate)
-
+```{r, eval = F}
```
Comment (dlebauer) [inline sensors/01-meteorological-data.Rmd]:
```suggestion
```{r, eval = FALSE}
```

File: sensors/06-list-datasets-by-plot.Rmd [hunk 9]
```diff
@@ -16,7 +16,7 @@ from the workbench or Globus.
 ## Getting started
 
 After installing terrautils, you should be able to import the `products` module.
-```{python}
+```{python eval = F}
```
Comment (dlebauer) [inline sensors/06-list-datasets-by-plot.Rmd]:
```suggestion
```{python eval = FALSE}
```

File: sensors/06-list-datasets-by-plot.Rmd [hunk 10]
```diff
@@ -38,12 +38,12 @@ sensor list and provides a list of names suitable for use in the
 To use this tutorial you will need to sign up for Clowder, have your 
 account approved, and then get an API key from the [Clowder web interface](https://terraref.ncsa.illinois.edu/clowder).
 
-```{python}
+```{python eval = F}
```
Comment (dlebauer) [inline sensors/06-list-datasets-by-plot.Rmd]:
```suggestion
```{python eval = FALSE}
```

File: sensors/06-list-datasets-by-plot.Rmd [hunk 11]
```diff
@@ -38,12 +38,12 @@ sensor list and provides a list of names suitable for use in the
...
 url = 'https://terraref.ncsa.illinois.edu/clowder/'
 key = 'ENTER YOUR KEY HERE'
 ```
 
-```{python}
+```{python eval = F}
```
Comment (dlebauer) [inline sensors/06-list-datasets-by-plot.Rmd]:
```suggestion
```{python eval = FALSE}
```

File: sensors/06-list-datasets-by-plot.Rmd [hunk 12]
```diff
@@ -85,7 +85,7 @@ dataset = get_file_listing(None, url, key, sensor, sitename,
 ```
 
 
-# Querying the API
+## Querying the API
```
Comment (dlebauer) [inline sensors/06-list-datasets-by-plot.Rmd]: maybe comment out this entire section?

Comment (jeanetteclark) [reply sensors/06-list-datasets-by-plot.Rmd]: I can, I didn't actually look at this chapter in terms of contentreally, just fixed the header levels and set `eval = FALSE` on all the chunks.

Comment (dlebauer) [reply sensors/06-list-datasets-by-plot.Rmd]: Its low priority ... @KristinaRiemer what do you think?

Comment (KristinaRiemer) [reply sensors/06-list-datasets-by-plot.Rmd]: I think it's worth having the curl method available in the tutorials, as long as the commands work.

**Pull request merged**